spec/features/students_page_spec.rb

```diff
@@ -68,6 +72,23 @@ def js_visit(path)
     expect(page).to have_content @user.username
   end
 
+
+  describe 'students view for user names' do
+    let(:user) { create(:user) }
+
+    it 'loads students view for user names without period' do
+       js_visit "/courses/#{@course.slug}/students/articles/#{@user.username}"
```

Comment:
Is it necessary to use `js_visit` and the `sleep 1` after it, or will these tests pass with just `visit`?

This is one of the only tests that has hard-coded `sleep` instead of relying on Capybara to automatically wait until the looked-for elements are on screen, and it might not be necessary anymore.

Reply:
> This PR should not include any changes to the schema.rb or Gemfile.lock files.

okay

Reply:
> Is it necessary to use `js_visit` and the `sleep 1` after it, or will these tests pass with just `visit`?
> 
> This is one of the only tests that has hard-coded `sleep` instead of relying on Capybara to automatically wait until the looked-for elements are on screen, and it might not be necessary anymore.

Thanks for the info. I am changing it